lib/strategies/header_strategy.rb

```diff
@@ -0,0 +1,22 @@
+module Phenotype 
+  class HeaderStrategy
+    attr_reader :header, :header_param, :env
+    def initialize(header: 'Sue')
+      @header = header
+    end
+
+    def get_version(env)
```

Comment:
Because we do not have access to the `env` object on initialization, only when rack calls `VersionedAppInstanc.call`